runtime/common/src/lib.rs

```diff
@@ -353,3 +354,13 @@ parameter_types! {
 	pub const OmnipoolLMOraclePeriod: OraclePeriod = OraclePeriod::TenMinutes;
 	pub const OmnipoolLMOracleSource: Source = OMNIPOOL_SOURCE;
 }
+
+// pallet dca
+parameter_types! {
+	pub StorageBondInNativeCurrency: Balance = 100 * UNITS;
+	pub MaxSchedulesPerBlock: u32 = 20;
+	pub SlippageLimitPercentage: Permill = Permill::from_percent(3);
+	pub MaxPriceDifference: Permill = Permill::from_percent(3);
```

Comment:
I would lower this to max 2% even 1.5% but this should really be configurable. ^ same as.